frontend/src/container/MetricsApplication/Tabs/util.ts

```diff
@@ -132,6 +158,42 @@ export function handleQueryChange(
 	};
 }
 
+export function useGetAPMToLogsQueries({
+	servicename,
+	filters,
+}: {
+	servicename: string;
+	filters?: TagFilterItem[];
+}): Query {
+	const finalFilters: TagFilterItem[] = [];
+	const { updateAllQueriesOperators } = useQueryBuilder();
+	let updatedQuery = updateAllQueriesOperators(
+		initialQueriesMap.logs,
+		PANEL_TYPES.LIST,
+		DataSource.LOGS,
+	);
+	const serviceName = {
+		id: 'service.name--string--resource--true',
+		dataType: DataTypes.String,
+		isColumn: true,
```

Comment:
@nityanandagohain shall we do `isColumn: false` here as well ? service.name won't always be a column right ?

Reply:
Since this is just filtering by `service.name`, we can keep `isColumn: false` but the backend should infer the correct type and implement automatically. I wonder how it worked when `isColumn: true` is defined as for most it is not defined as a column

Reply:
from the last conversation with @nityanandagohain i remember in case of `false` the qs intelligently identifies if its a column and filters correctly but `true` doesn't work if it's not a column. but there were some actions being taken at that time to improve this, not sure if it has been implemented or not. 

cc @nityanandagohain

Reply:
we can test with `isCoulmn: false` and close this conv if this works